native/acter/src/api/stream.rs

```diff
@@ -80,19 +94,195 @@ impl TimelineStream {
         matches!(self.room.state(), RoomState::Joined)
     }
 
-    pub async fn edit(&self, new_msg: String, original_event_id: String) -> Result<bool> {
+    // for integration test, not api
```

Comment:
maybe we should add a feature-flag for these non-regular-apis that we only enable for the integration tests?!?
```suggestion
    // for integration test, not api
    #[cfg(feature="testing")]
    #[doc(hidden)]
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1175/commits/33a25e307e0ab1cf1c9a03b262a3d61615effe15

Reply:
@bitfriend when I click the commit it tells me this is outside any branch or PR... when/how are you intending on proposing these to `main`?